### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by wocard2** [jankae/LCR#7] [General]
Thanks for this great project and for making it available! I was in the process of investigating what ADC to use for my own LCR meter when I stumbled across this project. Here's my variant of your LCR Meter (please excuse my inaptitude at taking pictures):

![LCR2](https://github.com/jankae/LCR/assets/52967668/e0afb341-2110-4b2a-bd95-bb080dcfe496)

I made some modifications to your design:

- the device is battery-powered (lipo battery) and charged via USB; see the battery voltage in the lower right corner
- the display uses a different controller (ST7789V2) and a different touch controller (FT6336)
- I use a different STM32 MCU with fewer pins (STM32F411RE)
- no isolation is used as the device is battery-powered
- as the DC/DC converter module is gone, the high bias voltage is created by a boost converter (MT3608)
- the boost converter is only turned on if the bias voltage exceeds some threshold
- the op amp is an LM7321 (R2Ri and R2Ro)

**Comment by jankae** [comment]
Awesome, thanks for sharing this! As far as I am aware, you are the first to succeed in replicating this project :)
